conf/core/ConfigOptions.asciidoc

```diff
@@ -1296,6 +1299,17 @@ If enabled, debug maps are written from inside loops where enabled. debug.maps.w
 be enabled for this option to be used. Enabling this option can create a lot of debug maps and slow
 down debugging significantly, so use with caution.
 
+=== differential.remove.linear.partial.matches.as.whole
+
```

Comment:
The main config option for the PR. Setting this to false, removes the geometric portions of the partial linear matches partially. Partial removal is the default.